### PR TITLE
feat(charts): Add support to add annotations for router during install

### PR DESCRIPTION
**Pull request by kmala**
ref https://github.com/deis/router/pull/288

### CODE REVIEW

File: charts/workflow/values.yaml [hunk 1]
```diff
@@ -165,6 +165,15 @@ registry-token-refresher:
 
 router:
   dhparam: ""
+  # Any custom router annotations(https://github.com/deis/router#annotations)
+  # which needs to applied can be specified as key values pair under this deployment_annotations key
```
Comment (mboersma) [inline charts/workflow/values.yaml]: I would rewrite this slightly:
`which need to be applied can be specified as key-value pairs under "deployment_annotations"`

File: charts/workflow/values.yaml [hunk 2]
```diff
@@ -165,6 +165,15 @@ registry-token-refresher:
...
+  deployment_annotations:
+    #<example-key>: <example-value>
+
+  # Any custom annotations for k8s services like http://kubernetes.io/docs/user-guide/services/#ssl-support-on-aws
+  # which needs to applied can be specified as key values pair under this service_annotations key
```
Comment (mboersma) [inline charts/workflow/values.yaml]: Same as above:
`which need to be applied can be specified as key-value pairs under "service_annotations"`

File: charts/workflow/values.yaml [hunk 3]
```diff
@@ -165,6 +165,15 @@ registry-token-refresher:
 
 router:
   dhparam: ""
+  # Any custom router annotations(https://github.com/deis/router#annotations)
+  # which needs to be applied can be specified as key-value pairs under "deployment_annotations"
```
Comment (mboersma) [inline charts/workflow/values.yaml]: `needs` should be `need` here and below. Otherwise this looks good to me.

**Comment by vdice** [comment]
Other than https://github.com/deis/workflow/pull/600#discussion_r87674794, LGTM

**Pull request merged**